edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Core.Test/commands/LoggingCommandFactoryTest.cs

```diff
@@ -11,8 +11,8 @@ namespace Microsoft.Azure.Devices.Edge.Agent.Core.Test.Commands
     using Microsoft.Extensions.Logging;
     using Moq;
     using Xunit;
-    using CommandMethodExpr = System.Linq.Expressions.Expression<System.Func<ICommandFactory, System.Threading.Tasks.Task<ICommand>>>;
-    using TestExecutionExpr = System.Func<LoggingCommandFactory, System.Threading.Tasks.Task<ICommand>>;
+    using CommandMethodExpr = System.Linq.Expressions.Expression<System.Func<Microsoft.Azure.Devices.Edge.Agent.Core.ICommandFactory, System.Threading.Tasks.Task<Microsoft.Azure.Devices.Edge.Agent.Core.ICommand>>>;
```

Comment:
I think I'd prefer a using statement and then avoid "Microsoft.Azure.Devices.Edge.Agent.Core" here.

Reply:
for type used in using statement; it doesn't reference to other using statement.  Therefore with stylecop checking, i need to provide it as full qualified name.